CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## X.X.X
+## 20.8.0 - 2022-08-01
```

Comment:
Looking at previous PRs, we should keep the X.X.X as the next release.
https://github.com/stripe/stripe-android/pull/5203/files
```suggestion
## X.X.X

## 20.8.0 - 2022-08-01
```